py/dml/ctree.py

```diff
@@ -4386,7 +4583,25 @@ def mkSubRef(site, expr, sub, op):
         else:
             raise EREF(site, sub, expr)
 
-    elif isinstance(expr, AddressOf) and op == '->':
+    if isinstance(expr, NonValue):
+        if op == '.':
+            if not site.dml_version() == (1, 2) and sub == 'len':
```

Comment:
`site.dml_version() != (1, 2)`?

Reply:
Oh. Sure. That's an inherited issue, but no reason not to address it.